onedal/dal.cpp

```diff
@@ -33,8 +33,11 @@ ONEDAL_PY_INIT_MODULE(polynomial_kernel);
 ONEDAL_PY_INIT_MODULE(sigmoid_kernel);
 
 /* algorithms */
-ONEDAL_PY_INIT_MODULE(svm);
+#if defined(ONEDAL_VERSION) && ONEDAL_VERSION >= 2023001ul
```

Comment:
Version is stored in INTEL_DAAL_VERSION which is defined in "services/library_version_info.h"
Also, it contains only digits, not `u1`

Reply:
Fixed